DoctorWho/DoctorWho.Db/Repositories/CallsRepository.cs

```diff
@@ -0,0 +1,29 @@
+﻿
+namespace DoctorWho.Db.Repositories
+{
+    public class CallsRepository
+    {
+        public static void CallViewEpisodesView()
+        {
+            var episodes = DoctorWhoCoreDbContext._context.ViewEpisodes.ToList();
+
+            Console.WriteLine(String.Format("{0, 5}|{1, 5}|{2, 5}|{3, 5}",
+                    "Doctor_Name", "Author_Name", "Companions", "Enemies"));
+            foreach (var epesode in episodes)
+            {
+                Console.WriteLine(String.Format("{0, 5}|{1, 5}|{2, 5}|{3, 5}",
+                   epesode.DoctorName, epesode.AuthorName, epesode.Companions, epesode.Enemies));
+            }
+        }
+        public static void CallfnCompanionsFunction(int id)
+        {
+            var companions = DoctorWhoCoreDbContext._context.Companions.Select(c => DoctorWhoCoreDbContext._context.CallFnCompanions(id)).FirstOrDefault();
+            Console.WriteLine(companions);
```

Comment:
it would make more sense for the function to return the results, not printing them, and rename the function to be more descriptive on what it's really returning.